src/os_posix.cc

```diff
@@ -279,6 +281,17 @@ bool OS::set_real_time(struct timespec* time) {
   FATAL("cannot set the time");
 }
 
+void OS::heap_summary_report(int max_pages, const char* marker, Process* process) {
+  const uint8* uuid = process->program()->id();
+  fprintf(stderr, "Out of memory: %08x-%04x-%04x-%04x-%04x%08x.\n",
```

Comment:
Here, I think it would be nicer with the process id that you can find in the table. It is super easy to get multiple processes with the same uuid.